Add more prose, fill in the WIPs (#55)
ember-learn/super-rentals-tutorial: src/chapters/01-orientation.md

```

```run:command hidden=true cwd=super-rentals
yarn test
git add tests/index.html
git commit --amend --no-edit
```

This should have created a new folder for us called `super-rentals`. We can navigate into it using the `cd` command.

```shell
$ cd super-rentals
```

> For the rest of the tutorial, all the commands should be run from within the `super-rentals` folder. This folder has the following structure:

Reviewer: ```suggestion
For the rest of the tutorial, all commands should be run within the `super-rentals` folder. This folder has the following structure:
```

```run:command lang=plain captureCommand=false
# Tree uses UTF-8 "non-breaking space" which gets turned into &nbsp;
# Somewhere in the guides repo's markdown pipeline there is a bug
# that further turns them into &amp;nbsp;

#[cfg(unix)]
tree super-rentals -a -I "node_modules|\.git" --dirsfirst | sed 's/\xC2\xA0/ /g'

#[cfg(windows)]
tree super-rentals /F
```

We will get to know the purposes of these files and folders as we go. For now, just know we will spend most of the time working within the `app` folder.

> Ember CLI comes with a lot of different commands for a variety of development tasks, such as the `new` command we saw earlier. It also comes with a *development server*, which we can launch with the `ember server` command:

Reviewer: ```suggestion
Ember CLI comes with a lot of different commands for a variety of development tasks, such as the `ember new` command that we saw earlier. It also comes with a *development server*, which we can launch with the `ember server` command:
```

ember server
```

> The development server is responsible for compiling our app and serving it to the browsers. It may take a while to boot up, but once it's up an running, open your favorite browser and head to http://localhost:4200. You should see the following welcome page:

Reviewer: ```suggestion
The development server is responsible for compiling our app and serving it to the browsers. It may take a while to boot up. Once it's up and running, open your favorite browser and head to http://localhost:4200. You should see the following welcome page:
```

```run:screenshot width=1024 retina=true filename=welcome.png alt="Welcome to Ember!"
visit http://localhost:4200/
```

> Zoey says...
>
> The `localhost` address in URL means that you can only access the development server from your local machine. If you would like to share your work to the world, you will have to *[deploy][TODO: link to deploy]* your app to the public Internet. Don't worry, we will cover that in Part 2 of the tutorial.

> You can exit out of the development server anytime but pressing `Ctrl + C` on your keyboard in the terminal which the server is running in. It is recommended that you run the development server in a new, dedicated terminal window. That way, you can keep it running in the background and won't have to stop and start the server everytime you want to run a command.

Reviewer: The sentences "It is recommended ... to run a command." might come across as a bit confusing to developers who are not used to Ember's live reload. I think we can more concisely describe the practice of having two tabs (terminals) open, one to run the server, and the other to type commands. 
```suggestion
You can exit out of the development server at any time by pressing `Ctrl + C` on your keyboard in the terminal where the server is running. We recommend having two terminal windows open: one to run the server in background, another to type Ember CLI commands.
```

> The development server has a feature called *live reload*, which monitors your app's files for changes, automatically re-compiles everything and refresh any open browser pages. This comes in really handy during development, let's give that a try!

Reviewer: ```suggestion
The development server has a feature called *live reload*, which monitors your app for file changes, automatically re-compiles everything, and refreshes any open browser pages. This comes in really handy during development, so let's give that a try!
```

As text on the welcome page pointed out, the source code for the page is located in `app/templates/application.hbs`. Let's try to edit that file and replace it with our own content:

```run:file:patch lang=handlebars cwd=super-rentals filename=app/templates/application.hbs
@@ -1,6 +1 @@
-{{!-- The following component displays Ember's default welcome message. --}}
-<WelcomePage />
-{{!-- Feel free to remove this! --}}
-
-{{outlet}}
-
+Hello World!!!
```

Soon after saving the file, your browser should automatically refresh and render our greetings to the world. Neat!

```run:screenshot width=1024 height=250 retina=true filename=hello-world.png alt="Hello World!!!"
visit http://localhost:4200/
```

When you are done experimenting, go ahead and delete the `app/templates/application.hbs` file. We won't be needing this for a while, so let's start afresh. We can add it back later when we have a need for it.

```run:command hidden=true cwd=super-rentals
git rm -f app/templates/application.hbs
```

> Again, if you still have your browser tab open, your tab should automatically reload into a blank page as you delete the file. This reflects the fact that we no longer have an application template in our app.

Reviewer: I'd write either `reload` (without "into") or `re-render`.
```suggestion
Again, if you still have your browser tab open, your tab will automatically re-render a blank page as soon as you delete the file. This reflects the fact that we no longer have an application template in our app.
```
